Task1/Task1/Program.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Numerics;
+
+namespace Task1
+{
+    class Program
+    {
+        private const int START_OBJECTS_AMOUNT = 3;
```

Comment:
https://stackoverflow.com/questions/242534/c-sharp-naming-convention-for-constants